applications/fishing-map/src/features/dataviews/dataviews.selectors.ts

```diff
@@ -263,3 +267,21 @@ export const selectActiveDataviews = createSelector(
     ...(activeContextAreasDataviews || []),
   ]
 )
+
+export const selectAvailableFishingDataviews = createSelector([selectAllDataviews], (dataviews) => {
+  return dataviews?.filter(
+    (d) =>
+      d.category === DataviewCategory.Fishing || DEFAULT_FISHING_DATAVIEW_ID === (d.id as number)
```

Comment:
Why the need for this part? :
> || DEFAULT_FISHING_DATAVIEW_ID === (d.id as number)

That specific dataview should also pass the `d.category === DataviewCategory.Fishing` test or am I missing something?

Reply:
Good catch, it was needed before using `selectAllDataviews` as input selector but now those dataviews are already in the list... so removed to keep it cleaner. Thanks!